Ghidra/Processors/PowerPC/data/languages/ppc_instructions.sinc

```diff
@@ -735,89 +735,109 @@
 
 #cmpw r0,r1		0x7c 00 08 00
 #cmpd r0,r1		0x7c 20 08 00  (64 bit mode)
-:cmp^DSIZE 	A,B			is OP=31 & CRFD=0 & BIT_22=0 & DSIZE & A & B & REG_A & REG_B & XOP_1_10=0 & BIT_0=0 
+:cmp^DSIZE A,B		is OP=31 & CRFD=0 & BIT_22=0 & DSIZE & A & B & REG_A & REG_B & XOP_1_10=0 & BIT_0=0 
 {
 	tmpA:$(REGISTER_SIZE) = REG_A;
 	tmpB:$(REGISTER_SIZE) = REG_B;
-	cr0 = ((tmpA s< tmpB) << 3) | ((tmpA s> tmpB) << 2) | ((tmpA == tmpB) << 1) | (xer_so & 1);
-	
+	setCrBit(cr0, 0, (tmpA s< tmpB));
```

Comment:
The manual says 0:2 are comparisons of result and 4th bit is SO of XER.  That logic does not match here, but at face value `setCrBit` does, but then it's helper macros are `3-bitIndex`.  Functionally this patch is not different,  I guess the question is why does it do `3-bitIndex` at all?